File: PYTHON/Tarea3_Recursividad/Potencia.py
def Potencia(base, exponente):
    if exponente == 0:
        return 1
    else:
        total = base * Potencia(base, exponente-1)
        return total


if __name__ == "__main__":

    base = int(input("Base: "))
    exponente = int(input("Exponente: "))
    print("Resultado: ",Potencia(base, exponente))

File: JAVA/Tarea3_Recursividad/Potencia.java
package JAVA.Tarea3_Recursividad;
import java.util.Scanner;

public class Potencia {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.println("\nBase: ");
        int base = sc.nextInt();

        System.out.println("Exponente: ");
        int exponente = sc.nextInt();

        System.out.println(potencia(base, exponente));
        
    }

    public static int potencia(int base, int exponente) {
        if(exponente == 0) return 1;
        return base *= potencia(base, exponente -1);
    }
   
}
